tests/base.py

```diff
@@ -106,3 +107,11 @@ def tearDown(self) -> None:
         self._conn.close()
         self._process.terminate()
         return super().tearDown()
+
+    @property
+    def gdb_version(self) -> Tuple[int, int]:
+        res = tuple(
+            map(int, re.search(r"(\d+)[^\d]+(\d+)", self._gdb.VERSION).groups())
```

Comment:
Is this for a version with just 2 numbers? We might want something like `r"(\d+)\D(\d+)"`. I don't think we need number `+` non-digits, and `\D` is the same as `[^\d]`

Reply:
TBH the intent is also to let things blow here because this is the same pattern we use and trust in gef. Having it fail here could hint issue with that in gef itself.